main.js

```diff
@@ -1884,33 +1898,41 @@ else {
     })
 
     app.whenReady().then(async () => {
-        checkWindowPosition(
+        var adjustedWindow = await checkWindowPosition(
```

Comment:
Why are we awaiting for this now?

Reply:
Because this needs to be completed before running

``` javascript
await createWindow()
```

at line 1939.
That was the whole reason the changes from `checkWindowPosition` weren't applied, and the reason why they were applied (even if wrong) if you halted the program ungracefully.

This resulted in the window opening outside of the monitor bounds.